README.md

```diff
@@ -1,16 +1,67 @@
-# Title
+# Building Cells: Getter and Setter Methods
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Builds a setter method to calculate computed properties
 
 ## Introduction
 
-## SWBAT 1
+Often in programming, you want to work with dynamic data. Just like
+in Ruby, you can _get_ and _set_ the values of variables. You can
+define a setter method execute when a specified property is being
+changed and have a getter method that returns a
+dynamically computed value, or reflects the status of an internal
+variable. There are a couple of ways to do this, so we will be
+building on the concept of _instance variables_ to illustrate this
+concept.
 
-## SWBAT 2
+
+## Builds a Setter Method to Calculate Computed Properties
+
+Let's look at an example that may be familiar:
+
+```js
+class Dog {
+  constructor(name, sex) {
+    this.name = name;
+  }
+
+  speak() {
+    return `${this.name} says woof!`
+  }
+}
+```
+Our class of `Dog` accepts 2 parameters, which are set each time a new
+instance of `Dog` is created.
+
+```js
+var buddy = Dog.new("Buddy", "male")
+
+buddy.speak() // returns 'Buddy says woof!'
+```
+
+Building on this concept, we're going to build our own shape calculator!
+- Create a class of `Circle`
+- `Circle` will accept 1 parameter with a setter method of `radius`
+- Define methods for `diameter`, `circumference`, and `area`.
+- Define a constant for `pi`
+- Given the `radius`, calculate the following in each corresponding method:
+  - `diameter`: `radius` x 2
```

Comment:
for each of these examples, we could have a `get`, the normal calculation, or a `set`, which uses the same property name, but takes an input and _sets_ radius, doing the math in reverse:

```
get diameter() {
return this.radius * 2
}
set diameter(diameter) {
this.radius = diameter/2
}
```